lifelines/utils.py

```diff
@@ -512,3 +513,36 @@ def qth_survival_time(q, survival_function):
 
 def median_survival_times(survival_functions):
     return qth_survival_times(0.5, survival_functions)
+
+
+def ridge_regression(X, Y, c1=0.0, c2=0.0, offset=None):
+    """
```

Comment:
I suggest expanding the docs a little. The wikipedia link was really helpful so it should be included in the docstring I think. Perhaps something like:

    Also known as 'Tikhonov regularization'. See http://en.wikipedia.org/wiki/Tikhonov_regularization
    for details. This solves a (possible ill-formed) minimization problem:
         (formula as before)